client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	English: i18n.translate( 'English' ),
+	'English lessons': i18n.translate( 'English lessons' ),
+	Enterprise: i18n.translate( 'Enterprise' ),
+	Entertainment: i18n.translate( 'Entertainment' ),
+	Entrepreneur: i18n.translate( 'Entrepreneur' ),
+	Entrepreneurship: i18n.translate( 'Entrepreneurship' ),
+	Environment: i18n.translate( 'Environment' ),
+	'Environment consultant': i18n.translate( 'Environment consultant' ),
+	Equestrian: i18n.translate( 'Equestrian' ),
+	Equity: i18n.translate( 'Equity' ),
+	Esteticien: i18n.translate( 'Esteticien' ),
+	Esthetician: i18n.translate( 'Esthetician' ),
+	Evangelisim: i18n.translate( 'Evangelisim' ),
+	Event: i18n.translate( 'Event' ),
+	'Event promotion': i18n.translate( 'Event promotion' ),
+	Events: i18n.translate( 'Events' ),
+	Excel: i18n.translate( 'Excel' ),
+	Exploration: i18n.translate( 'Exploration' ),
+	'Eye doctor': i18n.translate( 'Eye doctor' ),
+	'Face painting': i18n.translate( 'Face painting' ),
+	Faith: i18n.translate( 'Faith' ),
+	Family: i18n.translate( 'Family' ),
+	'Fan fiction': i18n.translate( 'Fan fiction' ),
+	Farm: i18n.translate( 'Farm' ),
+	Farming: i18n.translate( 'Farming' ),
+	Fashion: i18n.translate( 'Fashion' ),
+	'Fashion critic': i18n.translate( 'Fashion critic' ),
+	'Fashion design': i18n.translate( 'Fashion design' ),
+	Feelings: i18n.translate( 'Feelings' ),
+	Festival: i18n.translate( 'Festival' ),
+	'Fiction writing': i18n.translate( 'Fiction writing' ),
+	'Film production': i18n.translate( 'Film production' ),
+	Films: i18n.translate( 'Films' ),
+	Financial: i18n.translate( 'Financial' ),
+	'Financial consultant': i18n.translate( 'Financial consultant' ),
+	'Financial services': i18n.translate( 'Financial services' ),
+	'Fine art': i18n.translate( 'Fine art' ),
+	'First nations': i18n.translate( 'First nations' ),
+	Fishing: i18n.translate( 'Fishing' ),
+	Fitness: i18n.translate( 'Fitness' ),
+	Florist: i18n.translate( 'Florist' ),
+	'Flying club': i18n.translate( 'Flying club' ),
+	Food: i18n.translate( 'Food' ),
+	'Food critic': i18n.translate( 'Food critic' ),
+	Football: i18n.translate( 'Football' ),
+	'Freelance designer': i18n.translate( 'Freelance designer' ),
+	'Freelance developer': i18n.translate( 'Freelance developer' ),
+	'Freelance marketer': i18n.translate( 'Freelance marketer' ),
+	'Freelance services': i18n.translate( 'Freelance services' ),
+	'Freelance writer': i18n.translate( 'Freelance writer' ),
+	'Funeral services': i18n.translate( 'Funeral services' ),
+	Furniture: i18n.translate( 'Furniture' ),
+	Games: i18n.translate( 'Games' ),
+	Gaming: i18n.translate( 'Gaming' ),
+	Garments: i18n.translate( 'Garments' ),
+	Genealogy: i18n.translate( 'Genealogy' ),
+	Glamour: i18n.translate( 'Glamour' ),
+	Gloves: i18n.translate( 'Gloves' ),
+	God: i18n.translate( 'God' ),
+	Golf: i18n.translate( 'Golf' ),
+	Gospel: i18n.translate( 'Gospel' ),
+	'Government and politics': i18n.translate( 'Government and politics' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Government / Politics' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/translate-suggestions-about)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).